lib/rucio/core/rule.py

```diff
@@ -15,14 +15,14 @@
 
 import json
 import logging
-from collections.abc import Iterator
+from collections.abc import Iterable, Iterator
 from configparser import NoOptionError, NoSectionError
 from copy import deepcopy
 from datetime import datetime, timedelta
 from os import path
 from re import match
 from string import Template
-from typing import TYPE_CHECKING, Any, Callable, Optional, Type, TypeVar
+from typing import TYPE_CHECKING, Any, Callable, Literal, Optional, Type, TypeVar, Sequence
```

Comment:
Please be advised that `typing.Sequence` is deprecated and should be replaced with `collections.abc.Sequence`. See https://docs.python.org/3/library/typing.html#typing.Sequence. This applies to the other file too.

Reply:
Addressed here https://github.com/rucio/rucio/pull/6539